src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -105,7 +105,10 @@ class AdvertiserMinMdns : public ServiceAdvertiser,
     AdvertiserMinMdns() : mResponseSender(&GlobalMinimalMdnsServer::Server())
     {
         GlobalMinimalMdnsServer::Instance().SetQueryDelegate(this);
-        mResponseSender.AddQueryResponder(mQueryResponderAllocatorOperational.GetQueryResponder());
+        for (size_t i = 0; i < kMaxOperationalNetworks; ++i)
```

Comment:
Any reason not to make this a ranged for loop?

Reply:
That's fine - just a style difference. Does chip have a style guide that prefers ranged for?

Reply:
I think we prefer ranged for where possible, since it makes it harder to get the indexing wrong and usually leads to slightly more readable code in the loop.

Reply:
@cecille This is still using indexing?

Reply:
Missed one - sorry. Will update in next commit.

I'm half an half on ranged for - it makes the code lines more compact, but it does make it less obvious that you're touching a class member. Regardless, I don't really care enough to argue for index based. But I do think that if we're going to take a stance on style issues that we need to write down the preferences in some type of style guide. It's kind of a meta stability and velocity issue - making style changes necessitates re-testing the code and that testing may or may not be as thorough the next time and definitely causes a velocity reduction because you have to have another go at the presubmit. Getting the style consistent on the first go would increase velocity for all the devs. Because right now, the only guide I've heard about is the formatter and "when in Rome", which is different in this case.

Reply:
@andreilitvin - Can we start a basic guide with some of these common requests put down?

Reply:
Fixed in next commit.

Reply:
Going to resolve this for now, we can talk about style guide in the standup.